pkgs/development/libraries/fontconfig/make-fonts-cache.nix

```diff
@@ -13,6 +13,7 @@ runCommand "fc-cache"
   }
   ''
     export FONTCONFIG_FILE=$(pwd)/fonts.conf
+    export PATH=$PATH:${fontconfig}/bin
```

Comment:
(untested) This line may be unnecessary if you move fontconfig.bin from buildInputs to nativeBuildInputs.

Reply:
yeah, it looks like it's working too. I'll update that PR this evening.